setup.sh

```diff
@@ -334,6 +337,20 @@ COPYLEFT_LICENSE_INCLUDE = 'GPL* LGPL*'
     git pull --rebase origin eds-5.0.0
   fi
 
+  cd ${top_repo_dir}
+  qt5_dir=${top_repo_dir}/meta-qt5
+  if [ ! -d "${qt5_dir}" ]; then
+    # directory does not exist, create it
+    echo "Cloning meta-qt5 layer to ${top_repo_dir} directory from local cache"
+    git clone ${my_dl_dir}/meta-qt5-mirror.git meta-qt5
+    cd ${qt5_dir}
+    git checkout jansa/zeus
```

Comment:
I think it would be better to checkout zeus, not jansa/zeus. That will include also future patches to zeus.